Modules/Filtering/Smoothing/test/itkDiscreteGaussianImageFilterTest2.cxx

```diff
@@ -36,7 +42,9 @@ itkDiscreteGaussianImageFilterTestA(const char * inputFilename, float sigma, con
 
   auto filter = FilterType::New();
   filter->SetInput(reader->GetOutput());
-  filter->SetVariance(sigma);
+  filter->SetSigma(sigma);
+  filter->SetMaximumError(kernelError);
+  filter->SetMaximumKernelWidth(kernelWidth);
   filter->Update();
 
   using WriterType = itk::ImageFileWriter<ImageType>;
```

Comment:
using WriteImage would reduce amount of code.

Reply:
If possible I'd rather restrict changes to `itk::DiscreteGaussianImageFilter` and its test coverage to strictly necessary changes for this PR, such as exposing test parameters so that we can compare output more easily. We can consolidate test read/writes in a later PR.